README.md

```diff
@@ -8,6 +8,8 @@
 [![License](https://img.shields.io/badge/license-Apache%202-4EB1BA.svg)](https://www.apache.org/licenses/LICENSE-2.0.html)
 [![Go Report Card](https://goreportcard.com/badge/github.com/openyurtio/openyurt)](https://goreportcard.com/report/github.com/openyurtio/openyurt)
 [![codecov](https://codecov.io/gh/openyurtio/openyurt/branch/master/graph/badge.svg)](https://codecov.io/gh/openyurtio/openyurt)
+[![OpenSSF Scorecard](https://api.securityscorecards.dev/projects/github.com/openyurtio/openyurt/badge)](https://api.securityscorecards.dev/projects/github.com/openyurtio/openyurt)
+[![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/openyurt/badge)](https://bestpractices.coreinfrastructure.org/projects/openyurt)
```

Comment:
`https://bestpractices.coreinfrastructure.org/projects/openyurt` not Found

Reply:
You also need to manually create a project, refer to https://bestpractices.coreinfrastructure.org/en/projects